src/index.ts

```diff
@@ -564,7 +564,7 @@ const parsedSql = (sql: string): ParsedSql => {
           visitor.location
         ).trim();
         if (filter === "") {
-          return parsedSql(nextSql.replace(" WHERE ", ""));
+          return parsedSql(nextSql.replace(/ WHERE ?/gi, ""));
```

Comment:
Let's use the visitor location correctly for this, I guess if we have `SELECT " WHERE " FROM my_db WHERE slava = "the boss"`, this will fail